### PR TITLE
changeable projectile_damage_upgrade

### PR DESCRIPTION
**Pull request by derkachdaniil**
### Brief description of the changes:
value of ammo damage modifier now can be changed for each upgrade

### Tested Changes:
- [x] I've tested the changes locally or with people.
- [ ] I've not tested the changes.

### CODE REVIEW

**Comment by kajendra** [comment]
I might be mistaken, but the numbers for ammo modifiers should go in `maps/biter_battles_v2/tables.lua` as mentioned in the comment at the top of the file

File: maps/biter_battles_v2/functions.lua [hunk 1]
```diff
@@ -34,6 +39,27 @@ local balance_functions = {
 		global.combat_balance[force_name].shotgun = global.combat_balance[force_name].shotgun + get_upgrade_modifier("shotgun-shell")	
 		game.forces[force_name].set_ammo_damage_modifier("shotgun-shell", global.combat_balance[force_name].shotgun)
 	end,
+	["physical-projectile-damage-1"] = function(force_name)
```
Comment (kajendra) [inline maps/biter_battles_v2/functions.lua:42]: I believe these numbers should be in `maps/biter_battles_v2/tables.lua`

Comment (derkachdaniil) [reply maps/biter_battles_v2/functions.lua:42]: buff values can vary from function to function rn, so putting them to table will result in significantly bigger table, but can make it

Comment (derkachdaniil) [reply maps/biter_battles_v2/functions.lua:42]: PR#105

**Comment by kajendra** [comment]
Can be closed because I believe #105 is the new version of this.

**Comment by derkachdaniil** [comment]
#105 is untested - so i don't closing this for not

**Comment by 1pulse** [comment]
have to close this pr or not?

**Pull request merged**